.github/workflows/release.yml

```diff
@@ -0,0 +1,42 @@
+---
+name: Release
+
+on:
+  pull_request_target:
```

Comment:
I forgot why you use pull_request_target instead of pull_request

Reply:
To have it run in the context of the base (stable) branch instead of the pr (head) branch when triggered.